apps/in_hand_scanner/include/pcl/apps/in_hand_scanner/common_types.h

```diff
@@ -40,55 +40,45 @@
 
 #pragma once
 
-#include <cstdint>
-
+#include <pcl/geometry/triangle_mesh.h>
 #include <pcl/point_cloud.h>
 #include <pcl/point_types.h>
-#include <pcl/geometry/triangle_mesh.h>
 
-namespace pcl
-{
-  namespace ihs
-  {
-    struct PointIHS;
-    using CloudIHS = pcl::PointCloud<PointIHS>;
-    using CloudIHSPtr = CloudIHS::Ptr;
-    using CloudIHSConstPtr = CloudIHS::ConstPtr;
-  } // End namespace ihs
+#include <cstdint>
+
+namespace pcl {
+namespace ihs {
+struct PointIHS;
+using CloudIHS = pcl::PointCloud<PointIHS>;
+using CloudIHSPtr = CloudIHS::Ptr;
+using CloudIHSConstPtr = CloudIHS::ConstPtr;
+} // End namespace ihs
 } // End namespace pcl
 
 #include <pcl/apps/in_hand_scanner/impl/common_types.hpp>
 
-POINT_CLOUD_REGISTER_POINT_STRUCT (pcl::ihs::_PointIHS,
-                                   (float, x, x)
-                                   (float, y, y)
-                                   (float, z, z)
-                                   (float, normal_x, normal_x)
-                                   (float, normal_y, normal_y)
-                                   (float, normal_z, normal_z)
-                                   (float, rgb, rgb)
-                                   (float, weight, weight)
-                                   (unsigned int, age, age)
-                                   (std::uint32_t, directions, directions)
-                                  )
-POINT_CLOUD_REGISTER_POINT_WRAPPER (pcl::ihs::PointIHS, pcl::ihs::_PointIHS)
+POINT_CLOUD_REGISTER_POINT_STRUCT(
```

Comment:
This one got a bit harder to read - can we do anything about it?

Reply:
Yep, turning clang-tidy off for this lines. Some macros are a bit hard for automatic formatting ;-). We could check if newer version of Clang-Format have a way to define rules for this cases, but not sure.